lib/models/ui/ranking/ranking_element_details.dart

```diff
@@ -0,0 +1,44 @@
+import "package:flutter/foundation.dart";
+
+/// A class that stores data for each ranking UI element.
+/// The [userRank] is nullable to allow the current user to not have a rank.
+@immutable
+class RankingElementDetails {
+  const RankingElementDetails({
+    required this.userDisplayName,
+    required this.userUserName,
+    required this.centauriPoints,
+    required this.userRank,
+  });
+
```

Comment:
I think here we should have the user id too. We might use it for the avatar and the profile pop up.

Reply:
I've made some local progress on the user interface and it seems that the for the moment the `userId` is not necessary and I am able to display the ranking card information along with the pop up.

I will follow up on this discussion, if I have any update :)

Thank you :)

Reply:
Oh right. I just double-checked, I thought that `UserProfilePopUp` was taking a user id as parameter but apparently not :+1: